core/src/formatters/nova.ts

```diff
@@ -13,16 +13,11 @@ export interface NovaSystemMessage{
 }
 
 interface NovaMessagePart {
-    source?: {
-        type: "base64",
-        media_type: string,
-        data: string,
-    }, // only set for images
     text?: string // only set for text messages
     image?: {
         format: "jpeg" | "png" | "gif" | "webp",
         source: {
-            bytes: "base64",
+            bytes: string //"base64",
```

Comment:
are these changes intended?

Reply:
Yes, most of this code is copied from claude.ts where there was a fixed value property with the value "base64". Base64 is not a type, the property which contains the base64 is of type string. In my initial copy paste over from claude.ts I just had it as "base64", string is correct.